src/Code.gs

```diff
@@ -206,6 +224,26 @@ const ChatGPTApp = (function () {
         return this;
       };
 
+      /**
+      * Add an image to the chat. Will automatically get the description from gpt-4-vision-preview model and add the description as a message. 
+      * @param {string} imageUrl - The URL of the image to add.
+      * @param {string} fidelity - The level of fidelity for the image description
```

Comment:
value for fidelity should not be any string but either low or high
it's also optional and low by default / if not provided

Reply:
up